projects/fsastorefrontstyles/scss/base/checkboxes.scss

```diff
@@ -41,3 +41,15 @@ input[type='radio'] {
     }
   }
 }
+
+.checkbox-wrapper {
```

Comment:
Here I needed to override the default `tooltip-wrapper` styles to adjust them to this page and place info tooltip after checkbox label 'Select/Unselect'. Also, I reduced `z-index`, because when there is a global message displayed and the user scrolls screen, the info element was above the global message.

Reply:
We should try solving this without so much overriding. Just set:
```
.checkbox-wrapper {
  display: flex;
  .fs-tooltip {
    z-index: 2;
  }
}
```